### DISCUSSION TITLE
Welcome to Liger-Kernel Discussions!

### DISCUSSION THREAD

**Discussion by ByronHsu** [linkedin/Liger-Kernel#147] [Show and tell]
## 👋 Welcome!

Here is the place to show your work with Liger Kernel!!

**Comment by 2catycm** [comment]
Shall we apply liger kernel before lora or after lora?

**Comment by mumu029** [comment]
I'm a beginner. Can you please explain the difference between Triton code written for Liger kernels and Triton code generated by torch.compile? I feel like they seem the same, or maybe I'm mistaken.
